js/filter-panel.js

```diff
@@ -25,21 +28,25 @@ function FilterPanel(options) {
   this.$toggle.on('click', this.toggle.bind(this));
 
   this.filterSet = new FilterSet(this.$form).activate();
-  if (!_.isEmpty(this.filterSet.serialize())) {
-    this.show();
-  }
 
   this.adjust();
 }
 
 FilterPanel.prototype.adjust = function() {
```

Comment:
Is `adjust` an appropriate name for this method when the end result is showing or hiding the filter panel?  Should this be modifying the width of the panel instead since this appears to be checking for how much screen real estate is available to display the panel?

This is just my observation in processing this mentally, I haven't had a chance to run it yet for myself.